frontend/src/employee-frontend/components/voucher-value-decisions/VoucherValueDecisionDifferenceIcon.tsx

```diff
@@ -10,27 +10,58 @@ import { VoucherValueDecisionDifference } from 'lib-common/generated/api-types/i
 import RoundIcon from 'lib-components/atoms/RoundIcon'
 import Tooltip from 'lib-components/atoms/Tooltip'
 
-interface Props {
-  value: VoucherValueDecisionDifference
+interface IconsProps {
+  difference: readonly VoucherValueDecisionDifference[]
+  toggle?: (difference: VoucherValueDecisionDifference) => void
+  toggled?: VoucherValueDecisionDifference[]
+}
+
+export const VoucherValueDecisionDifferenceIcons = (props: IconsProps) => {
+  const { i18n, lang } = useTranslation()
+  return (
+    <>
+      {props.difference
+        .map((difference) => ({
+          value: difference,
+          label: i18n.valueDecisions.table.difference.value[difference],
+          labelShort:
+            i18n.valueDecisions.table.difference.valueShort[difference]
+        }))
+        .sort((a, b) => a.labelShort.localeCompare(b.labelShort, lang))
+        .map((item) => (
+          <VoucherValueDecisionDifferenceIcon
+            key={item.value}
+            item={item}
+            selected={props.toggled?.includes(item.value)}
+            toggle={props.toggle}
+          />
+        ))}
+    </>
+  )
+}
+
+interface IconProps {
+  item: {
+    value: VoucherValueDecisionDifference
+    label: string
+    labelShort: string
+  }
   selected?: boolean
-  toggle?: (difference: VoucherValueDecisionDifference) => () => void
+  toggle?: (difference: VoucherValueDecisionDifference) => void
 }
 
-export const VoucherValueDecisionDifferenceIcon = (props: Props) => {
-  const { i18n } = useTranslation()
+export const VoucherValueDecisionDifferenceIcon = (props: IconProps) => {
   const theme = useTheme()
   return (
-    <Tooltip
-      key={props.value}
-      tooltip={i18n.valueDecisions.table.difference.value[props.value]}
-      position="top"
-    >
+    <Tooltip key={props.item.value} tooltip={props.item.label} position="top">
       <RoundIcon
-        content={i18n.valueDecisions.table.difference.valueShort[props.value]}
+        content={props.item.labelShort}
         color={theme.colors.main.m1}
         size="L"
-        onClick={
-          props.toggle !== undefined ? props.toggle(props.value) : undefined
+        onClick={(_) =>
```

Comment:
You can just completely omit the parameter if it's not used
```suggestion
        onClick={() =>
```